File: Classes/CoreData/Objects/CDTag.m
//
//  CDTag.m
//  Dokie
//
//  Created by Ashamaz Shidov on 26/12/2019.
//  Copyright © 2019 Ashamaz Shidov. All rights reserved.
//

#import "CDTag.h"

@implementation CDTag

@dynamic text;
@dynamic document;

#pragma mark -
static NSString *_kText = @"text";
+ (NSString *)kText { return _kText; }

static NSString *_kDocument = @"document";
+ (NSString *)kDocument { return _kDocument; }

@end

File: Classes/CoreData/Objects/CDTag.swift
//
//  CDTag.swift
//  Dokie
//
//  Created by Ashamaz Shidov on 16.05.2020.
//  Copyright © 2020 Ashamaz Shidov. All rights reserved.
//

import CoreData

class CDTag : NSManagedObject {
    
    @NSManaged var text: String
    @NSManaged var document: CDDocument?

    static let kText = "text"
    static let kDocument = "document"

}
